changelogs/fragments/20240620-inventory-terraform_state-fix-issue-with-terraform-show.yaml

```diff
@@ -0,0 +1,5 @@
+---
+bugfixes:
```

Comment:
Is this a breaking_change?

Reply:
This change has no impact in the output display to the end user therefore cannot considered as `breaking change`